sender.go

```diff
@@ -828,3 +833,10 @@ func (b *buffer) WriteFloat(f float64) {
 	s := strconv.AppendFloat(a[0:0], f, 'G', -1, 64)
 	b.Write(s)
 }
+
+func (b *buffer) WriteBigInt(i *big.Int) {
+	// We need up to 256 bytes to fit an long256, including a sign.
```

Comment:
We're encoding a 256-bit unsigned integer into a hex string, so it should take up to 64 bytes, not 256 bytes.